sqlobject/dbconnection.py

```diff
@@ -827,6 +831,11 @@ def rollback(self):
                     inst.expire()
         self._makeObsolete()
 
+    def _send_event(self, signal):
+        sub_caches = [(sub[0], sub[1].allIDs()) for sub in self.cache.allSubCachesByClassNames().items()]
```

Comment:
What's the `sub` object?  Also, this is a good case for tuple unpacking

`[x, y.allIDs(), for x, y in self.cache.allSubCachesByClassNames().items()]`

Obviously better names would be preferred.

Reply:
Good point; I used chr for chr what they used a few lines up from here. Will refactor for clarity